src/Process/3dTilesProcessing.js

```diff
@@ -79,7 +79,51 @@ export function $3dTilesCulling(node, camera) {
     return false;
 }
 
+// Cleanup all 3dtiles|three.js starting from a given node n.
+// n's children can be of 2 types:
+//   - have a 'content' attribute -> it's a tileset and must
+//     be cleaned with cleanup3dTileset()
+//   - doesn't have 'content' -> it's a raw Object3D object,
+//     and must be cleaned with _cleanupObject3D()
+function cleanup3dTileset(layer, n) {
+    for (let i = 0; i < n.children.length; i++) {
+        // skip non-tiles elements
+        if (!n.children[i].content) {
+            n.children[i].traverse(_cleanupObject3D);
+        } else {
+            cleanup3dTileset(layer, n.children[i]);
+        }
+    }
+    if (n.dispose) {
+        n.dispose();
+    }
+    delete n.content;
+    layer.tileIndex.index[n.tileId].loaded = false;
```

Comment:
`loaded` is an attribute of the tile. By going through the tileIndex, you get the tile metadata (a portion of the tileset.json). I presume this is not intended?

Reply:
Fixed.